tests/test_cp_als.py

```diff
@@ -206,3 +206,28 @@ def test_cp_als_sptensor_zeros(capsys):
     capsys.readouterr()
     assert pytest.approx(output3["fit"], 1) == 0
     assert output3["normresidual"] == 0
+
+
+@pytest.mark.indevelopment
```

Comment:
No need for this mark. We don't use it anywhere and I think I've stripped them from a few of our test files. I should probably do a global remove at some point